docs/content/development/extensions.md

```diff
@@ -95,6 +95,7 @@ All of these community extensions can be downloaded using [pull-deps](../operati
 |kafka-emitter|Kafka metrics emitter|[link](../development/extensions-contrib/kafka-emitter.html)|
 |druid-thrift-extensions|Support thrift ingestion |[link](../development/extensions-contrib/thrift.html)|
 |druid-opentsdb-emitter|OpenTSDB metrics emitter |[link](../development/extensions-contrib/opentsdb-emitter.html)|
+|materialized-view-selection materialized-view-maintenance|Materialized View|[link](../development/extensions-contrib/materialized-view.html)|
```

Comment:
Would you please add a comma between them? Or perhaps itemizing them is better? Not sure it's possible in the table.

Reply:
Done.

Reply:
BTW, I kept the space as well for beautification reasons (Here, it creates two lines in the same cell).

Reply:
I'll keep it like that, but BTW it is possible to itemize:

|Name|Description|Docs|
|----|-----------|----|
|<ul><li>materialized-view-selection</li><li>materialized-view-maintenance</li></ul>|Materialized View|[link](../development/extensions-contrib/materialized-view.html)|

Reply:
Cool thanks!